src/checker.py

```diff
@@ -189,15 +227,24 @@ def get_outdated_external_data(self):
         return [
             data
             for data in self.get_external_data()
-            if data.state == ExternalData.State.BROKEN or data.new_version
+            if data.state == ExternalData.State.BROKEN or \
+               data.state == ExternalData.State.ADDED or \
+               data.state == ExternalData.State.REMOVED or \
+               data.new_version
         ]
 
     def _update_manifest(self, path, datas, changes):
         for data in datas:
-            if data.new_version is None:
+            if data.new_version is None and \
+               data.state != ExternalData.State.ADDED and \
+               data.state != ExternalData.State.REMOVED:
                 continue
 
             data.update()
+            if data.state == ExternalData.State.ADDED:
+                message = "Added {}".format(data.filename)
```

Comment:
I guess we'll find out whether, in practice, new translations are added/removed within the lifetime of a given Firefox release. I would expect not.

It would be nice if the commit message could read “Update Firefox to X.Y.Z” rather than “Update 94 modules”; and this would also make #17 easier because we would have already boiled the update down to the salient fact. But, as Voltaire probably didn't say, RESOLVED LATER is the friend of good!

Each time Firefox is updated, the commit message will be 180 lines long because every file will be both added (new version) and removed (old version) whereas the logical change is "Update Firefox to ____". I feel like it would be nice to be able to do

Reply:
Makes sense, I will have a think about it - we could probably use the module the updated data comes from to build the commit msg.